builtin/providers/aws/tagsRedshift_test.go

```diff
@@ -0,0 +1,54 @@
+package aws
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestDiffRedshiftTags(t *testing.T) {
+	cases := []struct {
+		Old, New       map[string]interface{}
+		Create, Remove map[string]string
+	}{
+		{
+			Old: map[string]interface{}{
+				"foo": "bar",
+			},
+			New: map[string]interface{}{
+				"bar": "baz",
+			},
+			Create: map[string]string{
+				"bar": "baz",
+			},
+			Remove: map[string]string{
+				"foo": "bar",
+			},
+		},
+		{
+			Old: map[string]interface{}{
+				"foo": "bar",
+			},
+			New: map[string]interface{}{
+				"foo": "baz",
+			},
+			Create: map[string]string{
+				"foo": "baz",
+			},
+			Remove: map[string]string{
+				"foo": "bar",
+			},
```

Comment:
Aren't these two cases completely the same? I think one of them should be different and maybe even test 2+ tags being added/removed at the same time.